internal/controller/controller.go

```diff
@@ -53,10 +53,21 @@ func (r *Route) Reconcile(ctx context.Context, req reconcile.Request) (reconcile
 	if err != nil {
 		return reconcile.Result{}, err
 	}
+	if len(route.ObjectMeta.OwnerReferences) > 0 {
+		for _, o := range route.ObjectMeta.OwnerReferences {
+			if o.Kind == "Ingress" {
+				log.V(5).Info("ignoring route owned by ingress", o.Name)
```

Comment:
**Unrelated:** I was surprised to learn that the default logging level we have for openshift-routes is `-v=5`... I wonder why that is 😅  It looks like most of log.V(5).Info output is very debug-centric (e.g., "started reconciling"). At some point later on, we should take a look at all the `log.V` instructions and see which ones are useful to the day-to-day operations and which ones are meant for debugging purposes.

Reply:
Let's fix that later.